packages/block-library/src/query/block.json

```
@@ -5,6 +5,7 @@
 	"title": "Query Loop",
 	"category": "theme",
 	"description": "An advanced block that allows displaying post types based on different query parameters and visual configurations.",
+	"keywords": [ "posts", "list", "blog", "blogs", "post types" ],
```

I think we should include "custom" alongside "post types", in case anyone is already familiar with that term.

```suggestion
	"keywords": [ "posts", "list", "blog", "blogs", "custom post types" ],
```